composer.json

```diff
@@ -4,9 +4,9 @@
     "type": "phpcodesniffer-standard",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4 || ^8.0",
-        "slevomat/coding-standard": "^6.3",
-        "squizlabs/php_codesniffer": "^3.5"
+        "php": ">=7.4",
```

Comment:
The concern with doing this is we're essentially saying we're compatible with all future versions of PHP. I think it's best to bump this as needed. The others are likely fine since I imagine future major versions will be backward compatible.